README.md

```diff
@@ -174,10 +174,10 @@ To create a highly available cluster of the Alertmanager the instances need to
 be configured to communicate with each other. This is configured using the
 `-mesh.*` flags.
 
-- `-mesh.hardware-address` string: MAC address, i.e. mesh peer ID (default "&lt;hardware-mac-address&gt;")
+- `-mesh.hardware-address` string: mesh peer ID (default "&lt;hardware-mac-address&gt;")
```

Comment:
I've thought about this before and was thinking that we should probably change this to `-mesh.peer-id` and have it default to the mac address, but not call it hardware address.